src/plugins/subsystems/drm/drm.sh

```diff
@@ -258,13 +258,14 @@ function get_available_connectors()
   local remote
   local port
   local find_conn_cmd
+  local connector_enabled
   declare -A cards
 
   flag=${flag:-'SILENT'}
 
   case "$target" in
     2) # LOCAL TARGET
-      cards_raw_list=$(find "$SYSFS_CLASS_DRM" -name 'card*' | sort -d)
+      cards_raw_list=$(find "$SYSFS_CLASS_DRM" -name 'card*-*' -exec printf '%s,' {} \; -exec cat {}/enabled \; -exec printf '\n' \; | sort -d)
```

Comment:
Add a small comment in this line that explains what's going on in the exec command. Also, use `--dictionary-order` instead of `-d` in the `sort` command. Check:
https://kworkflow.org/content/codingstyle.html#avoid-short-options